src/tools/setup-virtualenv.sh

```diff
@@ -22,12 +22,12 @@ virtualenv --python python2.7 $DIR
 . $DIR/bin/activate
 # older versions of pip will not install wrap_console scripts
 # when using wheel packages
-pip --log $DIR/log.txt install --upgrade 'pip >= 6.1'
+pip --disable-pip-version-check --log $DIR/log.txt install --upgrade 'pip >= 6.1'
```

Comment:
@dang what if pip does not support this option at that moment? see https://github.com/ceph/ceph/pull/10283#issuecomment-232434669